src/lib/network/clients/save_network.rb

```diff
@@ -191,13 +191,37 @@ def copy_from_instsys
       # but definitely not looking well ;-)
       NetworkAutoYast.instance.create_udevs if Mode.autoinst
 
+      copy_dhcp_info
+      copy_udev_rules
+      CopyConfiguredNetworkFiles()
+
+      # close and chroot back
+      WFM.SCRSetDefault(old_SCR)
+      WFM.SCRClose(new_SCR)
+
+      nil
+    end
+
+    def copy_dhcp_info
+      destdir = String.Quote(Installation.destdir)
+      # Wicked dhcp files
+      wicked_path = "/var/lib/wicked/"
+      wicked_files = ["duid.xml", "aid.xml", "lease*.xml"]
+      WFM.Execute(path(".local.bash"), "mkdir -p '#{wicked_path}'")
+      wicked_files.each do |file|
+        WFM.Execute(
```

Comment:
I may be wrong, but I would say that it is save to use `FileUtils.cp` with the `preserve` option, right? And it would be easier to avoid the loop in that way (as you won't need to worry about quoting the filenames).